Dockerfile

```diff
@@ -6,6 +6,10 @@ WORKDIR /
 COPY ./requirements.txt requirements.txt
 RUN pip install -r requirements.txt
 
+# create a new user and use it.
+RUN useradd -M -u 1001 nonrootuser
```

Comment:
@maximusunc should we fix the uid of the nonrootuser to be 1001? What if we don't specify the uid in the useradd command?

Reply:
According to Phil, it just needs to be an integer >= 1000. I don't think that's the case if we don't specify it.

Reply:
OK. Good to know. Agreed we need to test this out before deploying. I think we can deploy this in DILI development instance on ebcr0 for testing before deploying it to production instances.